src/components/services/AirshipInstance.tsx

```diff
@@ -14,6 +14,7 @@ export interface ShowErrorWarningOptions {
   // Report error to external bug tracking tool (ie. Bugsnag)
   trackError?: boolean
   tag?: string
+  onPress?: () => void | Promise<void>
```

Comment:
We want error reporting to be simple. It makes no sense to do `showError(e, { onPress: ... })`, since the `showError` function is responsible for the full UX.

If you need a red drop-down with custom UX, don't use `showError` for that. Just `Airship.show` the dropdown component itself.

Reply:
This still needs to be deleted.